README.md

```diff
@@ -1,70 +1,347 @@
-# Getting Started with Create React App
+<a name="readme-top"></a>
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+<div align="center">
+  <img src="./app-logo.png" alt="App logo">
+  <h1><b>Dota2 Insight Hub</b></h1>
 
-## Available Scripts
+</div>
 
-In the project directory, you can run:
+<!-- TABLE OF CONTENTS -->
 
-### `npm start`
+# 📗 Table of Contents
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+- [📖 About the Project](#about-project)
+  - [📹 Video Presentation](#video-presentation)
```

Comment:
- [ ]  There is a problem with your presentation link and its not opening. See the screenshot
<img width="908" alt="Screenshot 2023-10-14 at 6 29 28 PM" src="https://github.com/lily4178993/dota2-insight-hub/assets/44798044/abc95bd4-7e0a-4efa-a978-4a55ff716d2d">